src/biome/text/commands/helpers.py

```diff
@@ -145,7 +146,9 @@ def to_allennlp_params(self) -> Dict:
         for field, path in zip(data_fields, data_paths):
             if path:
                 allennlp_params[field] = path
-        if self.test_path:
-            allennlp_params[self.EVALUATE_ON_TEST_FIELD] = True
 
+        trainer_cfg = allennlp_params[self.TRAINER_FIELD]
+        if not trainer_cfg.get("type"):  # Just in case of default trainer
+            trainer_cfg["type"] = DefaultCallbackTrainer.__name__
+            allennlp_params[self.EVALUATE_ON_TEST_FIELD] = False
```

Comment:
Does this mean that if i want to use the `default` trainer (no callback trainer) i have to write:
```yaml
trainer:
    type: default
```
in my trainer yaml config?

Reply:
Exactly. Just in case of no trainer type is defined, the callback trainer is used as "default". If you set any type, it will be used as the trainer.

But, the idea is to be 100% compatible with the default ones configuration. So, have you a configuration sample of old configuration that doesn't work with the new trainer ?

Reply:
I had some trainer.yml files where i specified the `summary_interval`. I added this parameter to the `DefaultCallbackTrainer` and pushed the changes. If i encounter more cases i will either add them as well, or add this `type: default` line to the config file.